manifests/agent/ossec.pp

```diff
@@ -0,0 +1,21 @@
+#
+# = Class: zabbix::agent::ossec
+#
+# This module installs zabbix ossec sensor
+#
+class zabbix::agent::ossec (
+  $options                 = '',
+  $dir_zabbix_agentd_confd = $::zabbix::agent::dir_zabbix_agentd_confd,
+  $dir_zabbix_agent_libdir = $::zabbix::agent::dir_zabbix_agent_libdir,
+) inherits zabbix::agent {
+
+  file { "${dir_zabbix_agentd_confd}/ossec.conf" :
+    ensure  => file,
+    owner   => root,
+    group   => root,
+    content => template('zabbix/agent/ossec.conf.erb'),
+    notify  => Service['zabbix-agent'],
+    require => [ Package['wazuh-manager'] ],
```

Comment:
Is the package name applicable to all distros? Also, can this module be used with original OSSEC packages?

Reply:
You've got the point! It would be far better to replace package name with just a file name. That should keep the compatibility between all OSSEC/Wazuh versions, even the ones compiled from source.

Reply:
File dependency will only work if OSSEC/Wazuh module has file declaration in it.